Author: Adam Di Carlo <aph@debian.org>
Description: * docbook-4.0/dbpoolx.mod: Fixes some 3.1.7 references, that should be
4.0 FPIs.

<URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.2-6>
<URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.5-1>
--- a/docbook-4.0/dbpoolx.mod
+++ b/docbook-4.0/dbpoolx.mod
@@ -3354,7 +3354,7 @@
 
 <!-- Reference SGML Open Exchange Table Model -->
 <!ENTITY % tablemodel 
-  PUBLIC "-//Norman Walsh//DTD CALS Table Model XML V3.1.7//EN" 
+  PUBLIC "-//Norman Walsh//DTD CALS Table Model XML V4.0//EN" 
   "calstblx.dtd">
 ]]>
 
@@ -3388,7 +3388,7 @@
 <!ENTITY % tbl.entry.mdl "(%para.char.mix; | %tabentry.mix;)*">
 
 <!ENTITY % tablemodel 
-  PUBLIC "-//Norman Walsh//DTD Exchange Table Model 19960430 XML V3.1.7//EN"
+  PUBLIC "-//Norman Walsh//DTD Exchange Table Model 19960430 XML V4.0//EN"
   "soextblx.dtd">
 ]]>
 
